app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_words = command.split(" ")
+
+    if len(command_words) != 3:
+        return
```

Comment:
Consider providing feedback to the user if the command is invalid. Currently, the function returns silently, which might make it difficult for users to understand why the command didn't execute.